packages/docs/docs/tutorials/importing-sample-data.md

```diff
@@ -0,0 +1,61 @@
+
+Download one or both of the following files:
+
+| Data      | View                                                                                                                                                                      | Download                                                                                                    |
+| --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------- |
+| Patient 1 | [Patient 1 View Link](https://raw.githubusercontent.com/inferno-framework/inferno-reference-server/master/us-core-r4-resources/d831ec91-c7a3-4a61-9312-7ff0c4a32134.json) | [Patient 1 Download](https://drive.google.com/file/d/1bEyKSy55k9ZrrDLBj1NkHyL4ou75_eRX/view?usp=sharing)    |
+| Patient 2 | [Patient 2 View Link](https://raw.githubusercontent.com/inferno-framework/inferno-reference-server/master/us-core-r4-resources/e91975f5-9445-c11f-cabf-c3c6dae161f2.json) | [Patient 2 Download](https://drive.google.com/file/d/1Zj3EWeWj7-wP52CAZjSqzwCxZMlx6QZM/view?usp=share_link) |
+
+## Batch Upload Tool
+
+Log into Medplum and navigate to the [batch create](https://app.medplum.com/batch) tool. Upload the files you downloaded in the previous section.
+
+:::caution Note
+
+**Do not upload files twice.** If you aren't sure whether the upload worked, go to the [Patient](https://app.medplum.com/Patient) page. If you see a patient there - it worked.
+
+:::
+
+![Batch create tool](/img/tutorials/batch-create.png)
+
+The batch upload tool is a lightweight wrapper around the [batch/transaction api](https://www.hl7.org/fhir/http.html#transaction) and here is the documentation on how to [upload a batch using the SDK](/docs/sdk/classes/MedplumClient#batch).
+
+## Browse and Understand Data
+
+The dataset provided demonstrates how to represent many common data types. Click on the links provided to see the resources.
+
+| Data Type                                                                                                                                                                                                                                                                                                                                                                                   | Browse on Medplum                                                                                                                                                                                                                                                                                                                                      | Related Documentation                                                                                                                                                                        |
```

Comment:
Should refactor this out into a partial page, since it's used here and in the USCDI page: 
https://docusaurus.io/docs/markdown-features/react#importing-markdown

Reply:
Going to just link to to the tutorial to prevent dupes.